tuf/ngclient/updater.py

```diff
@@ -54,6 +56,7 @@
 from tuf.ngclient.fetcher import FetcherInterface
 
 logger = logging.getLogger(__name__)
+SUPPORTED_VERSIONS = ["1", "2", "3"]
```

Comment:
This should actually be `SUPPORTED_VERSIONS = [SPECIFICATION_VERSION]` for now.

Reply:
But I'm guessing that it'll have to change and be read from the disk?

Reply:
Got it.

Reply:
Can you push changes that address my comments before marking them as resolved? Makes it easier for me to track what we've been talking about.

Reply:
Sure. Will do that